runtime/src/test/java/io/quarkiverse/resteasy/problem/jackson/JacksonProblemDeserializerTest.java

```diff
@@ -0,0 +1,137 @@
+package io.quarkiverse.resteasy.problem.jackson;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import java.io.IOException;
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import com.fasterxml.jackson.core.JsonParseException;
+import com.fasterxml.jackson.databind.JsonMappingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.module.SimpleModule;
+
+import io.quarkiverse.resteasy.problem.HttpProblem;
+import io.quarkiverse.resteasy.problem.HttpProblemMother;
+
+class JacksonProblemDeserializerTest {
+
+    ObjectMapper mapper = new ObjectMapper();
+
+    @BeforeEach
```

Comment:
`@BeforeAll` might be enough enough...

Reply:
Then both `mapper` and `setup` method would have to be static